### PR TITLE
run plate 2 through cp and extract sc

### PR DESCRIPTION
**Pull request by jenna-tomkinson**
@gwaybio

This PR is for the run of plate 2 through CP pipeline with all CP modules (IC, seg, and feature extraction) and extract sc features.

### CODE REVIEW

File: CellProfiler_pipelines/Metadata/barcode_platemap_Plate2.csv [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Assay_Plate_Barcode,Plate_Map_Name
+2,platemap_NF1_CP_Plate2
```
Comment (gwaybio) [inline CellProfiler_pipelines/Metadata/barcode_platemap_Plate2.csv]: red symbol 🙃

File: CellProfiler_pipelines/Metadata/platemap_NF1_CP_Plate2.csv [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+F,7,F7,NF1,Null
+F,12,F12,NF1,Null
+G,1,G1,NF1,WT
+G,6,G6,NF1,WT
+G,7,G7,NF1,Null
+G,12,G12,NF1,Null
+H,1,H1,NF1,WT
+H,6,H6,NF1,WT
+H,7,H7,NF1,Null
+H,12,H12,NF1,Null
```
Comment (gwaybio) [inline CellProfiler_pipelines/Metadata/platemap_NF1_CP_Plate2.csv]: same here

**Review by gwaybio — APPROVED** [review]
Looks great! 😄

File: CellProfiler_pipelines/Metadata/barcode_platemap_Plate2.csv (hunk 1 above)
Comment (jenna-tomkinson) [reply CellProfiler_pipelines/Metadata/barcode_platemap_Plate2.csv]: Wait these happen on csvs too?! It is fixed :smile:

File: CellProfiler_pipelines/Metadata/platemap_NF1_CP_Plate2.csv (hunk 2 above)
Comment (jenna-tomkinson) [reply CellProfiler_pipelines/Metadata/platemap_NF1_CP_Plate2.csv]: Fixed!

**Pull request merged**